### PR TITLE
fix: custom form implementation

### PR DESCRIPTION
**Pull request by anhanh11001**
Details:
- Fetch custom form when ordering ticket. A custom field will contain information like this:
```
"attributes":{  
            "deleted-at":null,
            "field-identifier":"address",
            "type":"text",
            "is-required":false,
            "is-included":true,
            "form":"attendee",
            "is-fixed":false
         },
```
- Showing fields that is included in the custom forms
- Set required fields when form indicate required
- Apply data binding to Attendee View

Fixes: #1966

**Screenshots**
Example of applying all fields with firstname, lastname, email, github required
ezgif-4-95d1a4204f9a

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -43,7 +43,9 @@ android {
             resValue "string",  "FRONTEND_HOST", "eventyay.com"
         }
         debug {
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+
+//            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
```
Comment (nikit19) [inline app/build.gradle]: looks good. Just revert this

Comment (anhanh11001) [reply app/build.gradle]: Fixed

**Comment by anhanh11001** [comment]
Resolved conflicts

**Comment by iamareebjamal** [comment]
Database has changed to use 7.json now

**Comment by anhanh11001** [comment]
Updated

**Pull request merged**